cloudify_marketplace/resources/fix_cloudify_ip_configurations

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
```

Comment:
Aww, don't hurt bash's feelings or it'll all stop working!

But yeah, there are a few things we should probably support as part of the client- this, replacing certs (manager+broker), and resettings credentials (manager+broker).